api/errors.go

```diff
@@ -15,7 +15,7 @@ const (
 	ErrUserAlreadyExists       = Error("User already exists")
 	ErrInvalidUsername         = Error("Invalid username. White spaces are not allowed")
 	ErrAdminAlreadyInitialized = Error("An administrator user already exists")
-	ErrCannotRemoveAdmin       = Error("Cannot remove the default administrator account")
+	ErrCannotRemoveAdmin       = Error("Cannot remove this administrator account")
```

Comment:
See my comment below, I believe that this error definition is not needed anymore.